From: Evan Dandrea <ev@ubuntu.com>
Date: Wed, 25 May 2022 18:22:00 +0800
Subject: Use a framebuffer in the initramfs

Wubi disk image installs do a fair amount of work in the initramfs.
Last-Update: 2012-08-20
---
 scripts/build/lb_chroot_hacks | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/scripts/build/lb_chroot_hacks b/scripts/build/lb_chroot_hacks
index 06bfe50..d79ccd5 100755
--- a/scripts/build/lb_chroot_hacks
+++ b/scripts/build/lb_chroot_hacks
@@ -176,6 +176,12 @@ case "${LB_INITRAMFS}" in
 		;;
 esac
 
+case "${SUBPROJECT}" in
+	wubi)
+		UPDATE_INITRAMFS_OPTIONS="$UPDATE_INITRAMFS_OPTIONS FRAMEBUFFER=y"
+		;;
+esac
+
 if [ "${LB_INITRAMFS}" != "none" ]
 then
 	Chroot chroot "${UPDATE_INITRAMFS_OPTIONS} update-initramfs -k all -t -u"
